src/Microsoft.Health.Fhir.Shared.Core/Features/Operations/Reindex/ReindexUtilities.cs

```diff
@@ -54,7 +57,8 @@ public async Task ProcessSearchResultsAsync(SearchResult results, string searchP
                 var resourceElement = _deserializer.Deserialize(entry.Resource);
                 var newIndices = _searchIndexer.Extract(resourceElement);
                 var newIndicesHash = new HashSet<SearchIndexEntry>(newIndices);
-                var prevIndicesHash = new HashSet<SearchIndexEntry>(entry.Resource.SearchIndices);
+                var prevIndicesHash = entry.Resource.SearchIndices != null ?
```

Comment:
`entry.Resource.SearchIndices` will never have the SearchIndexEntry information since we don't have the de-serializer for it. We can simply this code (and the below if condition) by just removing this.

Reply:
That is true.  Will update this as part of the work item on possibly calculating a has of the given values.